components/GameForm.jsx

```diff
@@ -32,12 +44,35 @@ export default function GameForm({ onSubmit }) {
         id="gameDescriptionInput"
         name="description"
         required
+        defaultValue={formMode === "edit" ? selectedGame.description : ""}
       />
       <StyledButtonRow>
-        <button type="button" onClick={() => router.push("/")}>
-          Cancel
-        </button>
-        <button type="submit">Create</button>
+        {formMode === "add" && (
+          <>
+            <button type="button" onClick={() => router.push("/")}>
+              Cancel
+            </button>
+            <button type="submit">Create</button>
+          </>
+        )}
+        {formMode === "edit" && (
+          <>
+            <button
+              type="button"
+              onClick={() => {
+                router.push(`/${selectedGame.id}`);
+              }}
+            >
+              Cancel
+            </button>
+            <button
+              type="submit"
+              onClick={() => router.push(`/${selectedGame.id}`)}
+            >
+              Submit
+            </button>
+          </>
+        )}
```

Comment:
```suggestion
        <Link href={formMode === "edit" ? `/${selectedGame.id}` : "/"}>
          Cancel
        </Link>
        <button type="submit">
          {formMode === "edit" ? "Submit" : "Create"}
        </button>
```

- that is exactly the same. You only have to add `router.push(`/${gameToEdit.id}/`);` to the end of the 'handleEditGame' function in `_app.js` (like you did in the 'handleCreateGame').
- Please use the Link component here as well. You can style it exactly like the button if you want. 
- You can even make it more clean if you like: Since the form is either in 'edit' mode or not you can omit the 'add' mode. Then you only have to check e.g.: `{editMode ? "Submit" : "Create"}`.